README.md

```
@@ -119,10 +124,12 @@ Please keep in mind:
 More tips:
 - Guides to setup your development environment for [IntelliJ IDEA](https://github.com/dbpedia/extraction-framework/wiki/Setting-up-intellij-idea) or [Eclipse](https://github.com/dbpedia/extraction-framework/wiki/Setting-up-eclipse).
-- Get help with the [Maven build](Build-from-Source-with-Maven) or another form of [installation](Installation).
-- [Download](Downloads) some data to work with.
-- How to run [from Scala/Java](Run-from-Java-or-Scala) or [from a JAR](Run-from-a-JAR).
-- Having different troubles? Check the [troubleshooting page](Troubleshooting) or post on https://forum.dbpedia.org.
+- Get help with the [Maven build](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html) or another form of [installation](https://maven.apache.org/install.html).
+
+- [Downloads](https://www.dbpedia.org/resources/downloads/) some data to work with.
+- How to run from [Scala/Java](https://docs.scala-lang.org/tutorials/scala-with-maven.html) or from a [JAR.](https://docs.oracle.com/javase/tutorial/deployment/jar/run.html)
+
+- Having different troubles? Check the [troubleshooting page](https://maven.apache.org/users/getting-help.html)  or post on https://forum.dbpedia.org.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)